Kernel/Arch/aarch64/Processor.cpp

```diff
@@ -116,6 +116,14 @@ void Processor::flush_tlb_local(VirtualAddress, size_t)
     asm volatile("isb");
 }
 
+void Processor::flush_entire_tlb_local()
+{
+    asm volatile("dsb ishst");
+    asm volatile("tlbi vmalle1is");
```

Comment:
the name says local, but you're flushing the whole inner-shareable domain

Reply:
it seems this comment is more related to the `flush_tlb_local` method above (with a `FIXME`).

Reply:
No, this is `flush_entire_tlb_local`, this is flushing the entire TLB, but it's doing it globally, instead of locally, which is what the comment is saying.

Reply:
Ah good point. I changed it to a `tibi vmalle1`, which I think should only flush the caches of the core that execute the instruction, thus being 'local'.

Btw, I also tried to implement `Processor::flush_tlb_local` where I tried using `tibi vmalle1` (by va, all asid), but also `tibi vae1` (by va, specific asid), however both did not work. I'm not really sure why, but I guess flushing the whole TLB  cache is okay for now, we can always improve later :)

Reply:
a `TODO`/`FIXME` then, right? 😅